### PR TITLE
ZEPPELIN-297 Dependency should be loaded in pypsark

### PR DESCRIPTION
**Pull request by Leemoonsoo**
This PR fixes https://issues.apache.org/jira/browse/ZEPPELIN-297

* [x] fix, by set context classloader with classes from dependency loader when initializing py4j gatewayserver
* [x] test

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Ready to merge.

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

File: spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java [hunk 1]
```diff
@@ -125,6 +129,43 @@ private void createPythonScript() {
...
+      }
+    }
+
+    ClassLoader oldCl = Thread.currentThread().getContextClassLoader();
+    try {
+      URLClassLoader newCl = new URLClassLoader(urls, oldCl);
+      Thread.currentThread().setContextClassLoader(newCl);
+      createGatewayServerAndStartScript();
+    } catch (Exception e) {
+      throw new InterpreterException(e);
```
Comment (felixcheung) [inline spark/src/main/java/org/apache/zeppelin/spark/PySparkInterpreter.java]: add logger.error() here?

**Comment by felixcheung** [comment]
LGTM?

**Pull request closed without merging**